components/NewsGrid.vue

```diff
@@ -0,0 +1,45 @@
+
+<template>
+	<div class="news-grid">
+		<div v-for="row in groupedList" :key="row.id" class="row">
+			<div v-for="x in row" :key="x.id" class="api-content">
```

Comment:
Can we use more meaningful names instead of "x" and "row", something that would explain more clearly about what values they represent.

Reply:
how's `post` and `postRow`?

Reply:
This change has been implemented

Reply:
It just occurred to me that this is another artifact of trying to achieve the styling. Please correct me if I'm wrong. It looks like you are chunking the posts into a 2x2 array, then iterating over that to output a structure like:

```html
<div>
    <div>Post</div>
    <div>Post</div>
</div>
<div>
    <div>Post</div>
    <div>Post</div>
</div>
```

Which you are essentially using to create a two column layout of the News Items.

However, there may be a way to do this with styling, for example using [flex-box](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), [grid](https://css-tricks.com/snippets/css/complete-guide-grid/) or something else. Because we aren't worried about the styling, at the moment, let's simplify this by just outputting the posts and not worry about the chunking and columns for now.

Reply:
Okay, sounds good. I'll get started on that